spec/features/blog_spec.rb

```diff
@@ -52,82 +52,93 @@
-          visit blog_url_path
-
-          expect(page).to have_text(h1_text)
-
-          click_sign_in_with_google
-          fill_in('Public display name (optional)', with: Faker::Name.name)
-          click_on('Submit')
-
-          comment_text = 'I have something to say!'
-          fill_in('Write a comment...', with: comment_text)
-          click_on('Post')
-
-          expect(page).to have_css('.comment', text: comment_text)
-
-          Capybara.using_session('replier') do
-            MockOmniAuth.google_oauth2(email: existing_user_email)
+        context 'when the public_name of the user initially has a singe quote' do
+          let(:public_name) { "David O'Runger" }
 
+          it 'shows the blog article content and allows making comments, replying to comments, editing comments, and deleting comments' do
             visit blog_url_path
 
-            click_sign_in_with_google
-
-            click_on('Reply')
-            reply_text = 'I have something to say in reply!'
-            find_all(:fillable_field, 'Write a comment...')[1].send_keys(reply_text)
-
-            with_inline_sidekiq do
-              num_emails_before = ActionMailer::Base.deliveries.size
-              click_on('Reply')
-              # Wait for two emails: one to an admin and one to the user being replied to.
-              wait_for { ActionMailer::Base.deliveries.size }.to eq(num_emails_before + 2)
-            end
+            expect(page).to have_text(h1_text)
 
-            expect(page).to have_css('.comment', text: reply_text)
+            click_sign_in_with_google
+            fill_in('Public display name (optional)', with: public_name)
+            click_on('Submit')
 
-            open_email(new_user_email.downcase)
+            comment_text = 'I have something to say!'
+            fill_in('Write a comment...', with: comment_text)
+            click_on('Post')
 
-            expect(current_email).to be_present
-            expect(current_email).to have_text(<<~EMAIL.squish)
-              Someone has replied to a comment that you made at
-              https://davidrunger.com/blog/something-i-learned .
-              Comment content
-              I have something to say in reply!
-            EMAIL
+            expect(page).to have_css('.comment', text: comment_text)
 
-            click_on('Edit')
-            additional_text = ' Additionally!'
-            find_all(:fillable_field, 'Write a comment...')[1].send_keys(additional_text)
-            click_on('Update')
+            click_on('Edit your name', match: :first)
+            new_name = Faker::Name.name
+            fill_in('Public display name (optional)', with: new_name)
+            click_on('Submit')
 
-            expect(page).to have_css('.comment', text: "#{reply_text}#{additional_text}")
-          end
+            expect(page).to have_text("Post as #{new_name}")
```

Comment:
Most of the "diff" in this spec is just indenting things to accommodate the new `when the public_name of the user initially has a singe quote` context description. However, this part is new, because in addition to the hardcoded name with an apostrophe, I also want to continue using Faker to generate a name. That's how this bug was discovered, after all, so clearly there is some value in using the various names provided by Faker.